backend/clubs/views.py

```diff
@@ -5354,6 +5354,56 @@ def transfer(self, request, *args, **kwargs):
     def get_queryset(self):
         return Ticket.objects.filter(owner=self.request.user.id)
 
+    def give_tickets(self, order_info, cart, reconciliation_id):
+        """
+        Helper function that give the tickets to the user/buyer
+        and archive the transaction data
+        """
+
+        # At this point, we have validated that the payment was authorized
+        # Give the tickets to the user
+        tickets = (
+            cart.tickets.select_for_update()
+            .filter(holder=self.request.user)
+            .prefetch_related("carts")
+        )
```

Comment:
Ah, now I see why we're placing holds even on free tickets. Workaround might be to pass `check_for_hold` as a parameter (defaults to true)? cc @rm03

Reply:
nvm, thought about it some more. Holds are how we validate that a ticket was placed in a user's cart before they initiated checkout (even if it's not strictly needed here). Don't like the idea of adding a workaround to that.